src/trajectory/Interpolated.cpp

```diff
@@ -100,6 +100,17 @@ void Interpolated::evaluateDerivative(double _t, int _derivative,
     _tangentVector.setZero();
   }
 
+  if(_t==mWaypoints[0].t)
```

Comment:
I'd prefer to add a comment here. Because it is only when _t is exactly the same with mWaypoints[0].t , we do something special. Also, @mkoval do you think we shall not support evaluate Derivative when _t < mWaypoints[0].t?